docs/content/posts/2023-02-23-release-1.15.4.md

```diff
@@ -0,0 +1,152 @@
+---
+authors:
+- danny: null
+  name: Danny Cranmer
+date: "2023-02-23T17:00:00Z"
```

Comment:
Keep in mind that this date is ultimately what will be used when the blog post is published; so when you actually merge this, you should update it to the date of the release

Reply:
Yes, I usually update this with the actual release date prior to merge